### ISSUE TITLE
Show proper source in Sentry stack trace

### ISSUE THREAD

**Issue by mitchhentges** [james-proxy/james#226]
According to [Sentry](http://blog.getsentry.com/2015/10/29/debuggable-javascript-with-source-maps.html), it's possible to have stack traces properly show original source code/line numbers.

It would be much better than what we have right now, which just looks at the browserified bundle:

**Comment by tomitm** [comment]
Yeah, you basically need to create a release and upload the source maps through the API.

[`gulp-sentry-release`](https://github.com/ducdigital/gulp-sentry-release) looks like something we can drop into our gulp tasks.

**Comment by tomitm** [comment]
![image](https://cloud.githubusercontent.com/assets/4513759/15916405/945e710c-2da7-11e6-9b4e-a4a8a5cc844e.png)

Wow, if it wasn't for the awesome breadcrumbs, this exception would've been pretty useless.

**Comment by tomitm** [comment]
So, I did a little bit of preliminary digging on this.

- We're not actually generating JS sourcemaps right now - surprise! 🎊 
- My attempts to generate sourcemaps haven't been working well, possibly because we depend on electron/node's require loading at the moment. Might be related to why Chrome is showing files under _(no domain)_

Our build might need some work... maybe.
